File: src/main/java/com/codepine/api/testrail/model/Template.java
package com.codepine.api.testrail.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * TestRail template.
 */
public class Template {

    private int id;

    @JsonProperty(value = "is_default")
    private boolean isDefault;

    private String name;
}

File: src/main/java/com/codepine/api/testrail/model/Template.kt
package com.codepine.api.testrail.model

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * TestRail template.
 */
class Template {

    var id: Int = 0

    @JsonProperty(value = "is_default")
    var isDefault: Boolean = false

    var name: String? = null
}
